client-js/app/scripts/app.ts

```diff
@@ -23,10 +25,11 @@ window.addEventListener('WebComponentsReady', function() {
     page(e.detail.path);
   });
 
-  var socket = new registry.JsonRpcSocket('ws://localhost:9000/api/socket2');
+  var socket = new clientRpc.JsonRpcSocket('ws://localhost:9000/api/socket2');
+  socket.prefix = "org.labrad."
   var reg = new registry.RegistryServiceJsonRpc(socket);
+  var echo = new echoServer.EchoServiceJsonRpc(socket);
```

Comment:
yeah, I wasn't very clear. see #9 for what I was thinking, putting the prefix in a service base class, rather than the socket class. I also made it easier to cast return values to the expected types, which will help because in most situations not all methods have the same return type; the registry api currently does make all methods return the same type, but actually I think that should change, because we're transferring lots of extra data that's not in fact needed.